lib/autoproj/package_managers/apt_dpkg_manager.rb

```diff
@@ -8,23 +8,28 @@ class AptDpkgManager < ShellScriptManager
             def initialize(ws, status_file = "/var/lib/dpkg/status")
                 @status_file = status_file
                 @installed_packages = nil
+                @installed_versions = {}
                 super(ws, true,
                       %w{apt-get install},
                       %w{DEBIAN_FRONTEND=noninteractive apt-get install -y})
             end
 
-            def self.parse_package_status(installed_packages, paragraph)
+            def self.parse_package_status(installed_packages, paragraph, installed_versions: {})
```

Comment:
Try to stick to the current interface, that is `(installed_packages, installed_versions, paragraph)`

Reply:
This was for backward compatibility